apps/judicial-system/web/src/components/SelectCourtOfficials/SelectCourtOfficials.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { ValueType } from 'react-select'
+
+import { Box, Text, Tooltip, Select, Option } from '@island.is/island-ui/core'
+import { Case, User, UserRole } from '@island.is/judicial-system/types'
+
+import { selectCourtOfficials as strings } from './SelectCourtOfficials.strings'
+import { ReactSelectOption } from '../../types'
+import BlueBox from '../BlueBox/BlueBox'
+
+interface Props {
+  workingCase: Case
+  handleJudgeChange: (value: ValueType<ReactSelectOption>) => void
+  handleRegistrarChange: (value: ValueType<ReactSelectOption>) => void
+  users?: User[]
+}
+
+const SelectCourtOfficials: React.FC<Props> = (props) => {
+  const { workingCase, handleJudgeChange, handleRegistrarChange, users } = props
+  const { formatMessage } = useIntl()
+
+  const judges = (users ?? [])
+    .filter(
+      (user: User) =>
+        user.role === UserRole.JUDGE &&
+        user.institution?.id === workingCase.court?.id,
+    )
+    .map((judge: User) => {
+      return { label: judge.name, value: judge.id, judge }
+    })
+
+  const registrars = (users ?? [])
+    .filter(
+      (user: User) =>
+        user.role === UserRole.REGISTRAR &&
+        user.institution?.id === workingCase?.court?.id,
+    )
+    .map((registrar: User) => {
+      return { label: registrar.name, value: registrar.id, registrar }
+    })
+
+  const defaultJudge = judges?.find(
+    (judge: Option) => judge.value === workingCase?.judge?.id,
+  )
+
+  const defaultRegistrar = registrars?.find(
+    (registrar: Option) => registrar.value === workingCase?.registrar?.id,
+  )
+
+  return (
+    <>
+      <Box marginBottom={3}>
+        <Text as="h3" variant="h3">
+          {formatMessage(strings.title)}
+        </Text>
+      </Box>
```

Comment:
Would it make sense to have the tooltip in the `SectionHeading` since it accepts a tooltip?
```suggestion
      <SectionHeading title={formatMessage(strings.title)} tooltip={TODO} />
```